app/services/notifications.rb

```diff
@@ -51,4 +54,20 @@ def self.organization_data(organization)
       profile_image_90: organization.profile_image_90
     }
   end
+
+  def self.ancestor_data(comment)
+    comment.ancestors.includes(:user).map do |ancestor|
+      {
+        id: ancestor.id,
+        title: ancestor.title,
+        path: ancestor.path,
+        ancestry: ancestor.ancestry,
+        depth: ancestor.depth,
```

Comment:
`ancestry` is a column that shows the comment tree's path via IDs as a string, ex. `1/2/3/4`

`depth` is a method that shows the level of nesting as an integer, where the root comment is counted from 0.